config/patches/0004-Add-laziness-instructions-accumulate-readme.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/exercises/accumulate/README.md b/exercises/accumulate/README.md
```

Comment:
Could you perhaps merge this patch file with the previous one, so that there is only one patch file? That's the last :)

Reply:
yeah, just let me do some research on how to do that, and I will do it